src/components/add-mfa-method/index.njk

```diff
@@ -11,12 +11,16 @@
 {% set smsItem = {
     value: 'sms',
     text: 'pages.addMfaMethod.sms.title' | translate,
-    hint: 'pages.addMfaMethod.sms.hint' | translate
+    hint: { 
+        text: 'pages.addMfaMethod.sms.hint' | translate
+    }
 } %}
 {% set appItem = {
     value: 'app',
     text: 'pages.addMfaMethod.app.title' | translate,
-    hint: 'pages.addMfaMethod.app.hint' | translate
+    hint: {
+        text: 'pages.addMfaMethod.app.hint' | translate
```

Comment:
These is no value for this in the translation file. As per the design in figma, it doesn't look like there should be a hint for the auth app option, so I think the `hint` option can be removed here.